src/include/meta.h

```diff
@@ -37,6 +38,7 @@
  * Optimize comparisons against the metafile URI, flag handles that reference the metadata file.
  */
 #define WT_IS_METADATA(dh) F_ISSET((dh), WT_DHANDLE_IS_METADATA)
+#define WT_IS_LAS(dh) (dh->name_hash == WT_LAS_URI_HASHED)
```

Comment:
Nit: Should be fine in this case but generally, macro arguments should be wrapped in brackets. So `dh => (dh)`.